templates/product-analytics/moose/app/datamodels/models.ts

```diff
@@ -12,3 +12,37 @@ interface PageViewEvent {
   pathname: string;
   referrer: string;
 }
+
+export interface PageViewProcessed {
+  eventId: Key<string>;
+  timestamp: Date;
+  session_id: string;
+  locale: string;
+  location: string;
+  href: string;
+  hostname: string;
+  pathname: string;
+  referrer: string;
+  device_manufacturer: string;
+  browser_type: string;
+  browser_version: string;
+  os: string;
+  os_version: string;
+}
+
+export interface MooseActivityAugmented {
```

Comment:
Why did moose activity make it to the template? for testing?

Reply:
Yes- it should not be in there